x-pack/plugins/infra/public/alerting/log_threshold/components/alert_details_app_section/components/logs_history_chart.tsx

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import moment from 'moment';
+import React from 'react';
+import { Rule } from '@kbn/alerting-plugin/common';
+import { EuiPanel, EuiFlexGroup, EuiFlexItem, EuiTitle, EuiText, EuiSpacer } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { convertTo, TopAlert } from '@kbn/observability-plugin/public';
+import { AnnotationDomainType, LineAnnotation, Position } from '@elastic/charts';
+import { EuiIcon, EuiBadge } from '@elastic/eui';
+import { useFetchTriggeredAlertsHistory } from '../../../../../hooks/use_fetch_triggered_alert_history';
+import { type PartialCriterion } from '../../../../../../common/alerting/logs/log_threshold';
+import { CriterionPreview } from '../../expression_editor/criterion_preview_chart';
+import { PartialRuleParams } from '../../../../../../common/alerting/logs/log_threshold';
+
+const LogsHistoryChart = ({
+  rule,
+  alert,
+}: {
+  rule: Rule<PartialRuleParams>;
+  alert: TopAlert<Record<string, any>>;
+}) => {
+  // Show the Logs History Chart ONLY if we have one criteria
+  // So always pull the first criteria
+  const criteria = rule.params.criteria[0];
+  const dateNow = Date.now();
+  const dateLast30Days = Number(moment(dateNow).subtract(30, 'days').format('x'));
+  const { triggeredAlertsData } = useFetchTriggeredAlertsHistory({
+    features: 'logs',
```

Comment:
**nit:** consume the necessary features from the `AlertConsumers` dictionary to keep this working out of the box in case of definition changes.
```js
import { AlertConsumers } from '@kbn/rule-data-utils';

AlertConsumers.LOGS
```